### ISSUE TITLE
Errors encountered when testing Skosmos with Wave accessibility evaluation tool

### ISSUE THREAD

**Issue by tpalonen** [NatLibFi/Skosmos#642]
EU has a directive for online accessibility requirements and these requirements should be met by 23rd of September 2020. Wave Accessibility Evaluation Tool is a good way for testing whether some of these requirements are met.
http://wave.webaim.org

As tested for Finto service on 25th September 2017, the following errors and alerts were reported by Wave:
- Finto frontpage has 4 missing form labels, 2 orphaned form labels and 3 redundant title texts
- Finto concept page has 3 missing form labels, 1 missing field set and 1 redundant title text
- Finto enter search result page has 8 missing form labels, 2 orphaned form labels, 1 missing fieldset and alternative texts missing from each search result 

For further information on accessibility, see European Acessibility Act:
http://ec.europa.eu/social/main.jsp?catId=1202

**Comment by osma** [comment]
The UI will be rebuilt for Skosmos 3.0, taking proper consideration for accessibility. This list is already outdated.
Closing and archiving issue due to lack of activity. Please reopen if there are concrete reasons for reconsidering.
